tf2onnx/tfonnx.py

```diff
@@ -1835,6 +1836,7 @@ def where_op(ctx, node, name, args):
     "ReverseSequence": (reverse_op9, []),
     "Sign": (sign_op9, []),
     "Sinh": (direct_op, []),
+    "SparseSoftmaxCrossEntropyWithLogits": (sparse_softmax_cross_entropy_with_logits_op9, []),
```

Comment:
please also add a mapping for OneHot, maybe a new PR.

Reply:
will add another PR to avoid making the PR too complex.